rust/services/call/server_lib/src/handlers/v_call/types.rs

```diff
@@ -132,7 +132,7 @@ mod test {
     #[tokio::test]
     async fn invalid_to_address() {
         let call = Call::new(INVALID_ADDRESS, DATA, 0);
-        let actual_result = call.parse_and_validate(MAX_CALLDATA_SIZE);
+        let actual_result = call.parse_and_validate(MAX_CALLDATA_SIZE, 0);
```

Comment:
Since we will use `EVM_GAS_LIMIT` in all the other occurences of `call.parse_and_validate`, would it make sense to use it here as well? Bringing the tests closer to production behaviour?